lib/environment.rb

```diff
@@ -278,11 +278,14 @@ def parse_version(ver_str)
     # @return [Array<String>] raw version, major and minor number
     def get_version(user:)
       if opts[:version]
-        @major_version, @minor_version = parse_version(opts[:version])
-        return opts[:version], @major_version, @minor_version
+        _version = opts[:version]
+      else
+        _version = admin.cli_exec(:get, resource: "clusterversion", resource_name: "version", o: "jsonpath={.status.desired.version}")
       end
+      @major_version, @minor_version = parse_version(_version)
+      return _version, @major_version, @minor_version
 
-      raise "getting version of cluster as user not possible presently"
+      #raise "getting version of cluster as user not possible presently"
```

Comment:
We already set default version in `config/config.yaml` so we need to comment out that value if we want this to work OOB.

Also I think we better do like this:
```
if opst[:version]
  ...
elsif admin?
  ...
else
  raise "cluster version not set and getting without admin access not possible presently"
end
```

Reply:
@akostadinov  Updated.